src/main/java/frc/robot/LEDs/LEDIdleCommand.java

```diff
@@ -0,0 +1,63 @@
+package frc.robot.Util.LEDs;
+
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Loader.Intake;
+import frc.robot.Shooter.FiringPins;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDAnimation;
+import frc.robot.Util.LEDs.LEDSubsystem.LEDManager;
+
```

Comment:
Package Issues. These files do not exist in these packages so the imports fail.
There is no Util package.

Reply:
moved file into a new "util" folder and updated the package declaration.